packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -20,62 +22,79 @@ module.exports = {
   'Displays ajax error message in default': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
```

Comment:
This `1000` is the maximum time in millisecond to wait for `select` element to be visible.